packages/authentication/modules/access/useLogin/index.ts

```diff
@@ -17,6 +17,7 @@ import { useForm } from 'react-hook-form'
 import AuthApi from '../../../services/auth'
```

Comment:
You shoudln't be here, this file was removed. Login is now handled in useAllAuthLogin. Did you base this off the correct branch? `BA-1786-fe-mfa-setup-fe-library`